packages/AreaChart/src/AreaChart.tsx

```diff
@@ -139,7 +143,19 @@ const AreaChart: React.FunctionComponent<AreaChartProps> = ({
     const DEFAULT_SKELETON_WIDTH = 24;
     const DEFAULT_SKELETON_HEIGHT = 8;
 
-    Settings.defaultLocale = locale;
+    const localeMapping: Record<string, string> = {
+        "en": "en-US",
+        "fr": "fr-CA"
+    };
+    const providerLocale = useLocale();
+    let actualLocale = providerLocale.locale;
```

Comment:
Can this synthax be used? we do this a lot in orbit

```
    // in the props deconstruction
    locale: localeProp,
...
   const providerLocale = useLocale();
   const locale = localeMapping[locale] ?? providerLocale 

```

Reply:
I didn't do that because the user can set either en of en-US for example.
Unfortunately we supported just en before, but in the component, I want to change it to use en-US.
but I want to support them setting en-US as well (which they may have been doing anyway... Hard to tell in OV's code to be honest..